integration_tests/ci/sample.profiles.yml

```diff
@@ -1,13 +1,13 @@
 
-# HEY! This file is used in the Google Ads integrations tests with CircleCI.
+# HEY! This file is used in the dbt package integrations tests with Buildkite.
```

Comment:
Oops. Do we need to go back and update these or will they be batch-updated? Forgot to do this.

Reply:
I will fix this in the migrations, so you guys don't need to worry. I figured that if we're releasing this onto main right now, we should update this so that it's up to date.